test/react/16.5/test/getTestDeps.js

```diff
@@ -0,0 +1,7 @@
+import enzyme from 'enzyme'
```

Comment:
this seems like leftover from refactoring fro menzyme to react-testing-library, should I remove all those files (they are in each, or most, directories)?

Reply:
Oh, looks like that removed the `react/${version}/test/` directory, which is needed. Can you add empty `.keep` files back to keep the dirs in git, or add something to the install script to create them?

Reply:
Fixed